tests/php/tests/includes/test_class.wp-job-manager-usage-tracking-data.php

```diff
@@ -391,6 +391,61 @@ public function test_jobs_featured() {
 		$this->assertEquals( $published + $expired, $data['jobs_featured'] );
 	}
 
+	/**
+	 * Tests that get_usage_data() returns the correct number of job listings
+	 * posted by guests.
+	 *
+	 * @since 1.30.0
+	 * @covers WP_Job_Manager_Usage_Tracking_Data::get_usage_data
+	 */
+	public function test_jobs_by_guests() {
+		$published_by_guest = 3;
+		$expired_by_guest   = 2;
+
+		// Create published listings.
+		$this->factory->job_listing->create_many( $published_by_guest, array(
+			'post_author' => '0',
+		) );
+
+		// Create expired listings.
+		$this->factory->job_listing->create_many( $expired_by_guest, array(
+			'post_author' => '0',
+			'post_status' => 'expired',
+		) );
+
+		// Create guest listings with other statuses.
+		$statuses = array( 'future', 'draft', 'pending', 'private', 'trash' );
+		foreach ( $statuses as $status ) {
+			$params = array(
+				'post_author' => '0',
+				'post_status' => $status,
+			);
+
+			if ( 'future' == $status ) {
+				$params['post_date'] = '3018-02-15 00:00:00';
+			}
+
+			$this->factory->job_listing->create( $params );
+		}
+
+		// Create listings with other author.
+		$all_statuses = array_merge( $statuses, array( 'publish', 'expired' ) );
+		$author_id    = $this->factory->user->create();
+		foreach ( $all_statuses as $status ) {
+			$params = array(
+				'post_author' => $author_id,
+				'post_status' => $status,
+			);
+
+			if ( 'future' == $status ) {
```

Comment:
Was this run through the linter? I would have expected it to complain about `==`.

Reply:
It wasn't. I ran it and fixed the errors.